packages/flutter/lib/src/widgets/animated_child_switcher.dart

```diff
@@ -53,17 +148,69 @@ class AnimatedChildSwitcher extends StatefulWidget {
   /// [duration].
   final Widget child;
 
+  /// The duration over which to perform the cross fade using [FadeTransition].
+  final Duration duration;
+
   /// The animation curve to use when fading in the current widget.
   final Curve switchInCurve;
 
   /// The animation curve to use when fading out the previous widgets.
```

Comment:
The animation curve to use when transitioning the previous [child] out.

Reply:
Done.